frontend/leaderboard/src/Main.re

```diff
@@ -1,14 +1,17 @@
-type newBlock = {protocolState: string};
+let blockDirectory =
+  ([%bs.node __dirname] |> Belt.Option.getExn |> Filename.dirname)
+  ++ "/src/blocks/";
 
-external unsafeJSONToNewBlock: Js.Json.t => newBlock = "%identity";
+let files = blockDirectory |> Node.Fs.readdirSync;
 
-let files = Node.Fs.readdirSync("blocks");
-Array.iter(
-  _file => {
-    let fileContents = Node.Fs.readFileAsUtf8Sync("block.json");
-    let blockDataJson = Js.Json.parseExn(fileContents);
-    let newBlock = unsafeJSONToNewBlock(blockDataJson);
-    print_endline(newBlock.protocolState);
-  },
-  files,
-);
+let blocks =
+  Array.map(
+    _file => {
```

Comment:
prepending underscore usually means the variable is unused, feel free to remove the _ now that you're using it